File: python3_solution_set0/1929. Concatenation of Array.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        
        
        ans = []
        for x in range(len(nums)):
            ans.append(nums[x])
            
            
        for y in range(len(nums)):
            ans.append(nums[y])
            
            
        return ans
        

File: go_solution_set1/1929. Concatenation of Array.go
func getConcatenation(nums []int) []int {
    ans := []int{}
    for x:=0; x<len(nums);x++{
        ans = append(ans,0)
        ans = append(ans, 0)
    }
    n := len(nums)
    for x:=0; x<len(nums);x++{
        ans[x] = nums[x]
        ans[x+n] = nums[x]
    }


    return ans
}
